Nef_S2/examples/Nef_S2/CMakeLists.txt

```
@@ -6,17 +6,77 @@ project( Nef_S2_Examples )
 
 cmake_minimum_required(VERSION 3.1)
 
-find_package(CGAL QUIET)
+if(POLICY CMP0053)
+  # Only set CMP0053 to OLD with CMake<3.10, otherwise there is a warning.
+  if(NOT POLICY CMP0070)
+    cmake_policy(SET CMP0053 OLD)
+  else()
+    cmake_policy(SET CMP0053 NEW)
+  endif()
+endif()
+if(POLICY CMP0071)
+    cmake_policy(SET CMP0071 NEW)
+endif()
+
+find_package(CGAL)
+find_package(Qt5Widgets REQUIRED)
+find_package(Qt5OpenGL REQUIRED)
```

You should search for the package Qt5 instead of `Qt5Widgets` and `Qt5OpenGL`:
```suggestion
find_package(Qt5 COMPONENT Widgets OpenGL)
```

In our Travis configuration, we only pass `Qt5_DIR` to cmake, and not `Qt5Widgets_DIR`. That is why the Travis checks failed.